src/data/data-gig-detail.html

```diff
@@ -1,12 +1,18 @@
 <link rel="import" href="../../bower_components/polymer/polymer.html">
 
 <link rel="import" href="../../bower_components/polymerfire/firebase-document.html">
+<link rel="import" href="data-gigs-list.html">
 
 
 <dom-module id="data-gig-detail">
 <template>
   <firebase-document id="document" app-name="polygigs"
     path="/gigs/[[gigId]]" data="{{raw}}"></firebase-document>
+
+  <!--<template is="dom-if" if="{{!document.online}}">-->
```

Comment:
Isn't it `navigator.onLine`? Even with that change it may not work like you'd expect because that doesn't abide by the Polymer binding protocol (meaning `navigator` does not fire `online-changed` events).

`<app-indexeddb-mirror>` has an `online` property that would seem to work. That's available in `app-storage`.